Auto find cur (#205)
metricly/metricly-docs: content/integrations/AWS-Integration/aws-cur.md

13. Select **Next**.
![final-steps](/images/aws-cur/final-steps.png)
14. Review your configurations.
16. Select **Review and Complete** to create the Cost and Usage Report (CUR).

It can take up to a few hours for data to populate in the S3 bucket.

## 2. Update your AWS Integration in CloudWisdom

If you have just created your [IAM role][1], **wait 2-5 minutes** for AWS to finalize its creation before proceeding to these steps. This ensures the new role has the correct s3 access permissions when added to CloudWisdom.

1. From the side navigation menu, select **Integrations**.
2. Select the **Amazon Web Services** card.

> 3. Ensure both **Cost Explorer API** and **Detailed Billing** are enabled.

Reviewer: Cost Explorer isn't part of CUR and the toggle name just changed to "Cost & Usage Reports"
```suggestion
3. Ensure **Cost & Usage Reports** is enabled.
```

> 4. Scroll to the **Detailed Billing Settings** section.

Reviewer: Renamed to "Cost & Usage Reports"
```suggestion
4. Scroll to the **Cost & Usage Reports settings** section.
```

5. Select a **Report Path Prefix** from the dropdown.
![cur-lookup](/images/aws-cur/cur-lookup.png)
5. Select **Save**.  

> {{% notice tip %}}

Reviewer: I do like this tip a lot. Could we still have the old setup steps just in case we don't have the CUR access?

5. Select a **Report Path Prefix** from the dropdown.
![cur-lookup](/images/aws-cur/cur-lookup.png)
5. Select **Save**.  

{{% notice tip %}}

If the **Report Path Prefix** lookup is not available (and is instead a freeform text input), make sure you are using the most recent IAM role available. You can create the latest IAM role using the our [CloudFormation script](/integrations/aws-integration/aws-cloudformation-installation/).
You also have the option to manually enter the **S3 Bucket Name** (chosen in section 1, step 7) and **Report Path Prefix** (created in section 1, step 8).
{{% /notice %}}

---

## Cost & Usage Report Best Practices

The following best practices ensure optimal Cost & Usage report setup.

### Shared Master Billing Account: How to Grant Limited Access

Some customers store their billing files in a shared master billing account and need to grant CloudWisdom restricted access to one specific S3 bucket. You can use the [master billing account permissions](/integrations/aws-integration/aws-iam-installation/#master-billing-account-permissions) to create a limited read-only IAM role when configuring your AWS integration with CloudWisdom.

> For customers using a shared S3 bucket: Virtana only reads the costs for accounts that we monitor; we discard all data for any unrelated accounts.

Reviewer: ```suggestion
For customers using a shared S3 bucket: CloudWisdom only reads the costs for accounts that we monitor; data for unrelated accounts is discarded.
```

### Request CUR Setup for Resold Resources

> You may not have access to your billing preferences if you have purchased AWS services through a reseller. You must contact your reseller and request to be set up with an S3 bucket to store your CUR data. You can then use [this minimal read-only IAM role](http://localhost:1313/integrations/aws-integration/aws-iam-installation/#alternative-create-a-custom-policy-with-minimal-permissions-read-only-role) to allow CloudWisdom access.

Reviewer: ```suggestion
You may not have access to your billing preferences if you have purchased AWS services through a reseller. You must contact your reseller and request to be set up with an S3 bucket to store your CUR data. This [minimal read-only IAM role](http://localhost:1313/integrations/aws-integration/aws-iam-installation/#alternative-create-a-custom-policy-with-minimal-permissions-read-only-role) can be used to allow CloudWisdom read-only access to a single S3 bucket containing the billing files that help to power our cost reporting features.
```
